CODEOWNERS

```diff
@@ -3,6 +3,7 @@
 # review when someone opens a pull request.
 *       @Aniruddh25 @seantleonard @aaronburtle @abhishekkumams @RubenCerna2079 @ravishetye @rohkhann @neeraj-sharma2592 @sourabh1007
 
+/docs @jerrynixon
```

Comment:
`/docs` houses any design documents we end up tracking in source control. Otherwise, public docs maintained in separate docs repo. Are you planning on adding more docs here?